### ISSUE TITLE
cannot tabulate on ngx-intl-tel

### ISSUE THREAD

**Issue by bouyaahmedsami** [webcat12345/ngx-intl-tel-input#425]
I'm trying to use your library for our project so I referred your demo page.

They said that it is keyboard accessible, however, we cannot tabulate the component,and have access to it with keyborad arrows .
Do you have any backlog item to make it compliance with accessibility in near future ?
Do you have any tips , or solution to have the component accessible with keyboard ?
@pasevin
@ambersz

Thank you

**Comment by cmckni3** [comment]
Duplicate of #421
